fhir-persistence/src/test/java/com/ibm/fhir/persistence/search/test/AbstractSearchQuantityTest.java

```diff
@@ -38,42 +38,41 @@ public void testSearchQuantity_Quantity() throws Exception {
         assertSearchReturnsSavedResource("Quantity", "25|http://unitsofmeasure.org|s");
         assertSearchReturnsSavedResource("Quantity", "25||s");
         assertSearchReturnsSavedResource("Quantity", "25");
-        
+
         // https://gforge.hl7.org/gf/project/fhir/tracker/?action=TrackerItemEdit&tracker_item_id=19597
         assertSearchReturnsSavedResource("Quantity", "25||sec");
-        
-        assertSearchDoesntReturnSavedResource("Quantity", "24.4999||s");
 
-//        assertSearchReturnsSavedResource("Quantity", "24.5||s");
-//        assertSearchReturnsSavedResource("Quantity", "25.4999||s");
+        assertSearchDoesntReturnSavedResource("Quantity", "24.4999||s");
+        assertSearchDoesntReturnSavedResource("Quantity", "24.5||s");
+        assertSearchDoesntReturnSavedResource("Quantity", "25.019||s");
         assertSearchDoesntReturnSavedResource("Quantity", "25.5||s");
     }
-    
+
     @Test
     public void testSearchToken_Quantity_or() throws Exception {
         assertSearchReturnsSavedResource("Quantity", "10||a,25||s,30||z");
     }
-    
+
     @Test
     public void testSearchToken_Quantity_escaped() throws Exception {
         assertSearchReturnsSavedResource("Quantity", "25|http://unitsofmeasure.org|s");
         assertSearchDoesntReturnSavedResource("Quantity", "25|http://unitsofmeasure.org\\||s");
     }
-    
+
     @Test
     public void testSearchQuantity_Quantity_chained() throws Exception {
         assertSearchReturnsComposition("subject:Basic.Quantity", "25|http://unitsofmeasure.org|s");
         assertSearchReturnsComposition("subject:Basic.Quantity", "25||s");
-        
+
         // DSTU2 does not say if this is allowed or not, but we do not support it.
         // In more recent versions, they clarified that it should work:  https://build.fhir.org/search.html#quantity
-//        assertSearchReturnsComposition("Quantity", "25");
-        
+        //  assertSearchReturnsComposition("Quantity", "25");
+
         // I think this should return the resource but it currently doesn't.
         // https://gforge.hl7.org/gf/project/fhir/tracker/?action=TrackerItemEdit&tracker_item_id=19597
```

Comment:
HL7 moved from "GForge" to JIRA.  The link should now be c
```suggestion
        // https://jira.hl7.org/browse/FHIR-19597
```